include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -233,6 +233,14 @@ struct __atomic_ref : sycl::atomic<_AtomicType, _Space>
 };
 #endif // _ONEDPL_SYCL2023_ATOMIC_REF_PRESENT
 
+template <typename DataT, sycl::access::mode AccessMode, int Dimensions = 1>
+using __local_accessor =
+#if _ONEDPL_LIBSYCL_VERSION >= 60000
+    sycl::local_accessor<DataT, Dimensions>;
```

Comment:
For additional information about class sycl::local_accessor please see
https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_interface_for_local_accessors